.github/workflows/ci.yml

```diff
@@ -16,16 +16,9 @@ jobs:
           echo "$JAVA_HOME_17_X64/bin" >> $GITHUB_PATH
           echo "JAVA_HOME=$JAVA_HOME_17_X64" >> $GITHUB_ENV
         shell: bash
-      - name: install Crow
+      - name: Setup environment
         run: |
-          wget https://github.com/CrowCpp/Crow/releases/download/v1.0%2B5/crow-v1.0+5.deb
-          sudo apt install ./crow-v1.0+5.deb
-      - name: Install ROS2
-        uses: lf-lang/lingua-franca/.github/actions/setup-ros2@master
-      - name: Setup ROS2
-        run: | 
-          source /opt/ros/*/setup.bash
-          printenv >> "$GITHUB_ENV"
+          source ./utils/scripts/setup-env.bash
```

Comment:
Sourcing our setup script does not help as it does not define any environment variables. We have to define the setup scrips in /opt/ros/ installed by ROS. Also they need to be piped to "$GITHUB_ENV" in order to be available in the later steps.

Reply:
Would adding `printenv >> "$GITHUB_ENV"` to `setup-env.bash` be the right fix?

Reply:
No, because `setup-env.bash` does not source `/opt/ros/*/setup.bash`. The C++ tests fail because they don't find the ROS2 installation. And they don't find the ROS2 installation because the environment variables set by `/opt/ros/*/setup.bash` are not available.

I noticed that our new setup script modifies bashrc to source `/opt/ros/*/setup.bash`. This might work in the playground, but it does not work here as we don't run the tests through bash. So I think we would need to resurrect the complete "Setup ROS2" step as it was before. We could also try to reduce it to
```
name: Setup ROS2
  run: printenv >> "$GITHUB_ENV"
  shell: bash
```
This should work if the step loads the modified bashrc.

Reply:
> No, because `setup-env.bash` does not source `/opt/ros/*/setup.bash`.

But I had added it. Maybe @axmmisaka removed it again?

Reply:
> ```
> name: Setup ROS2
>   run: printenv >> "$GITHUB_ENV"
>   shell: bash
> ```
> 
> This should work if the step loads the modified bashrc.

I see now. Yes, that's a good suggestion. Let's try that.